testsuite/features/support/commonlib.rb

```diff
@@ -728,3 +728,46 @@ def api_unlock
     file.flock(File::LOCK_UN)
   end
 end
+
+# Function to fetch the current list of events
+def fetch_event_history(hostname)
+  output, _code = get_target('server').run("spacecmd -u admin -p admin system_listeventhistory #{hostname}", check_errors: true)
+  events = []
+  current_event = {}
+
+  output.split("\n").each do |line|
```

Comment:
Single quotes maybe.

Reply:
I think it doesn't recognize it if using single quote because it's not just a string.